var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -161,7 +161,7 @@ class Trilinos(CMakePackage):
             description='Compile with Piro')
     variant('phalanx',      default=False,
             description='Compile with Phalanx')
-    variant('rol',          default=False,
+    variant('rol',          default=True,
```

Comment:
Is this change really needed for xsdk? I don't think I ever needed this.

Or is this needed for dealii?

Reply:
> Is this change really needed for xsdk? I don't think I ever needed this.

that's needed to avoid concretization issues in Spack, otherwise `xsdk` won't concretize.